README.md

```diff
@@ -104,39 +102,45 @@ Usage:
 
 ```sh
 BEAKER_vagrant_box_tree=$vagrant_boxes_dir \
-BEAKER_box__puppet="simpci/server-6.2.0-RC1.el7-CentOS-7.0.x86-64" \
+BEAKER_box__puppet="simpci/server-6.2.0-RC1.el7-CentOS-6.9.x86-64" \
   bundle exec rake beaker:suites[default]
 ```
 
 #### `upgrade`
 
-The **`upgrade`** suite validates the SIMP user guide's [General Upgrade Instructions for incremental upgrades][u0] by upgrading an older version of SIMP.  It:
+The **`upgrade`** suite validates the SIMP user guide's [General Upgrade
+Instructions for incremental upgrades][u0] by upgrading an older version of
+SIMP.  It:
 
-1. Uploads a newer SIMP version's `.iso` file
+1. Uploads a newer SIMP version's `.iso` file from the host filesystem into the
+   puppetserver SUT.
+  - Uploads any `*.noarch.rpm` files from the local project's directory into
+    the puppet server's yum repository before running `unpack_dvd`
 2. Runs `unpack_dvd`
-3. Runs `puppet agent -t`
-
-
-[u0]: https://github.com/simp/simp-doc/blob/8277eab/docs/user_guide/Upgrade_SIMP/General_Upgrade_Instructions.rst#incremental-updates
+3. Runs `yum update -y`
+4. Runs `puppet agent -t`
 
 Usage:
 
 ```sh
 BEAKER_vagrant_box_tree=$vagrant_boxes_dir \
```

Comment:
$vagrant_boxes_dir is not explained anywhere.  I'm assuming its the directory named 'boxes' in the vagrant VM directory tree?

Reply:
It's an example variable (like `$PATH_TO_VAGRANT_BOXES_DIRTREE` in the previous example) to stand in form the directory tree created by `rake vagrant:publish:local` and  `rake simp:packer:matrix`, as described by the README.

`$PATH_TO_VAGRANT_BOXES_DIRTREE` is more wordy but seems less confusing, so I'll switch this to that.

Reply:
:ballot_box_with_check:  in d4a221f

Reply:
:ballot_box_with_check::ballot_box_with_check: c058865
^^Added diagrams for what env vars point to what